modules/nw-route-specific-annotations.adoc

```diff
@@ -128,31 +128,31 @@ WebSocket connections to timeout frequently on that route.
 ----
 metadata:
   annotations:
-    haproxy.router.openshift.io/ip_whitelist: 192.168.1.10
+    haproxy.router.openshift.io/ip_allow: 192.168.1.10
 ----
 
 .A route that allows several IP addresses
 [source,yaml]
 ----
 metadata:
   annotations:
-    haproxy.router.openshift.io/ip_whitelist: 192.168.1.10 192.168.1.11 192.168.1.12
+    haproxy.router.openshift.io/ip_allow: 192.168.1.10 192.168.1.11 192.168.1.12
 ----
 
 .A route that allows an IP address CIDR network
 [source,yaml]
 ----
 metadata:
   annotations:
-    haproxy.router.openshift.io/ip_whitelist: 192.168.1.0/24
+    haproxy.router.openshift.io/ip_allow: 192.168.1.0/24
 ----
 
 .A route that allows both IP an address and IP address CIDR networks
 [source,yaml]
 ----
 metadata:
   annotations:
-    haproxy.router.openshift.io/ip_whitelist: 180.5.61.153 192.168.1.0/24 10.0.0.0/8
+    haproxy.router.openshift.io/ip_allow: 180.5.61.153 192.168.1.0/24 10.0.0.0/8
```

Comment:
These examples should all use `haproxy.router.openshift.io/ip_allowlist` rather than `haproxy.router.openshift.io/ip_allow`.